lib/FusionInventory/Agent/Task/Inventory/Virtualization/Xen.pm

```diff
@@ -8,7 +8,7 @@ use FusionInventory::Agent::Tools;
 our $runMeIfTheseChecksFailed = ["FusionInventory::Agent::Task::Inventory::Virtualization::Libvirt"];
 
 sub isEnabled {
-    return canRun('xm');
+    return canRunOK('xm') || canRunOK('xl');
 }
```

Comment:
@ddurieux : testing the version is unfortunately not relevant, because the choice of the toolstack is made through configuration.